packages/editor/src/lib/hooks/useCoarsePointer.ts

```diff
@@ -17,14 +17,28 @@ export function useCoarsePointer() {
 			return
 		}
 		if (window.matchMedia) {
-			const mql = window.matchMedia('(pointer: coarse)')
-			const handler = () => {
+			const mql = window.matchMedia('(any-pointer: coarse)')
+
+			const handler = (coarse?: boolean) => {
 				editor.updateInstanceState({ isCoarsePointer: !!mql.matches })
+				if (coarse !== undefined) {
+					editor.updateInstanceState({ isCoarsePointer: coarse })
+				}
```

Comment:
i think this should go before the above and it should be an if / else type deal.
or, you could go simpler, as in:

```suggestion
				editor.updateInstanceState({ isCoarsePointer: coarse ?? !!mql.matches })```